### PR TITLE
Added BoneArena, BoneType, Skeleton

### PR DESCRIPTION
**Pull request by TheButlah**

### CODE REVIEW

File: src/color.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    def_color!(MAROON, 128, 0, 0);
+    def_color!(YELLOW, 255, 255, 0);
+    def_color!(OLIVE, 128, 128, 0);
+    def_color!(LIME, 0, 255, 0);
+    def_color!(GREEN, 0, 128, 0);
+    def_color!(AQUA, 0, 255, 255);
+    def_color!(TEAL, 0, 128, 128);
+    def_color!(BLUE, 0, 0, 255);
+    def_color!(NAVY, 0, 0, 128);
+    def_color!(FUCHIA, 255, 0, 255);
```
Comment (TheDevMinerTV) [inline src/color.rs]:
```suggestion
    def_color!(FUCHSIA, 255, 0, 255);
```

File: src/model/bone_kind.rs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    ThighL,
+    ThighR,
+    AnkleL,
+    AnkleR,
+    FootL,
+    FootR,
+
+    UpperArmL,
+    UpperArmR,
+    ForeArmL,
```
Comment (TheDevMinerTV) [inline src/model/bone_kind.rs]: This should be lowercase, no?

File: src/model/bone_kind.rs [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    ForeArmR,
```
Comment (TheDevMinerTV) [inline src/model/bone_kind.rs]: Same here

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: src/model/bone_kind.rs (hunk 2 above)
Comment (TheButlah) [reply src/model/bone_kind.rs]: I don't think so? Not sure

**Comment by TheButlah** [comment]
Addressed @TheDevMinerTV comments

**Pull request merged**